addons/website/migrations/12.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,153 @@
+# NOTHING TO DO
+
+website      / website                  / auth_signup_uninvited (selection): NEW selection_keys: ['b2b', 'b2c']
+website      / website                  / country_group_ids (many2many) : NEW relation: res.country.group
+website      / website                  / google_maps_api_key (char)    : NEW
+website      / website                  / social_default_image (binary) : NEW attachment: True
+website      / website                  / social_instagram (char)       : NEW
+website      / website                  / specific_user_account (boolean): NEW
+website      / website                  / theme_id (many2one)           : NEW relation: ir.module.module
+# NOTHING TO DO
+
+website      / website                  / social_facebook (char)        : not related anymore
+website      / website                  / social_facebook (char)        : is now stored
+website      / website                  / social_github (char)          : not related anymore
+website      / website                  / social_github (char)          : is now stored
+website      / website                  / social_googleplus (char)      : not related anymore
+website      / website                  / social_googleplus (char)      : is now stored
+website      / website                  / social_linkedin (char)        : not related anymore
+website      / website                  / social_linkedin (char)        : is now stored
+website      / website                  / social_twitter (char)         : not related anymore
+website      / website                  / social_twitter (char)         : is now stored
+website      / website                  / social_youtube (char)         : not related anymore
+website      / website                  / social_youtube (char)         : is now stored
+# DONE: post-migration: filled from res.company
+
+website      / website                  / name (char)                   : now required
+website      / website                  / company_id (many2one)         : now required, default = function
+# DONE: post-migration: filled fields in case they are empty
+
+website      / website.menu             / parent_left (integer)         : DEL
+website      / website.menu             / parent_path (char)            : NEW 
+website      / website.menu             / parent_right (integer)        : DEL 
+# DONE: post-migration: called _parent_store_compute method
+
+website      / website.page             / header_color (char)           : NEW
+website      / website.page             / header_overlay (boolean)      : NEW 
+website      / website.page             / is_published (boolean)        : NEW
+# NOTHING TO DO
+
+website      / website.page             / website_id (many2one)         : is now stored
+website      / website.page             / website_ids (many2many)       : DEL relation: website
+# DONE: post-migration: create a copy page for each website in case for pages with more than one website
```

Comment:
In v12 works differently in multiwebsite. I tested it when I did the `apply_multiwebsite` script.

Reply:
I have been testing this and the mechanism is:

- If there's a view with `website_id` specified and the same key as the master one, use this.
- If not, use the master one without website but with that key.
- On a fresh DB, you only have master views, but on first touch you make on a page, then a copy is populated with specific website.

Thus, I still prefer the approach of copying all website views for each website, being one or several. Why? Because that way, we can do a second step, which is reset master views to v12 original content, and leave the others untouched, and we will have both of best worlds.